src/expr/src/scalar/func/impls/string.rs

```diff
@@ -921,3 +921,62 @@ sqlfunc!(
         panic!("{}", a)
     }
 );
+
+/// QuoteIdent is transliterated from postgres/src/backend/utils/adt/ruleutils.c
```

Comment:
Can you include the function name to make it easier to find (I'm assuming it's `quote_identifier`).

Is this true though? It looks like we just call `mz_sql_parser::ast::Ident::from`, maybe this comment belongs on top of that function?